pkg/service/oidc4ci/oidc4ci_service_initiate_issuance.go

```diff
@@ -65,11 +65,14 @@ func (s *Service) InitiateIssuance(
 		Scope:                              req.Scope,
 		OpState:                            req.OpState,
 		ClaimData:                          req.ClaimData,
-		UserPinRequired:                    req.UserPinRequired,
 		State:                              TransactionStateIssuanceInitiated,
 		WebHookURL:                         profile.WebHook,
 	}
 
+	if req.UserPinRequired {
+		data.OtpPin = s.pinGenerator.Generate()
+	}
+
 	if data.GrantType == "" {
 		data.GrantType = defaultGrantType
```

Comment:
what is `defaultGrantType ` here?

Reply:
its constant, defaultGrantType    = "authorization_code" @sudeshrshetty